tests/cursor.rs

```diff
@@ -87,3 +87,49 @@ fn test_tailing_cursor() {
     // 15 results.
     assert_eq!(25, guard.join().expect("Thread failed"));
 }
+
+#[test]
+fn test_batch_cursor() {
+    let uri      = Uri::new("mongodb://localhost:27017/").unwrap();
+    let pool     = Arc::new(ClientPool::new(uri, None));
+    let client   = pool.pop();
+    let database = client.get_database("rust_test");
+
+    const TEST_COLLECTION_NAME: &str = "test_batch_cursor";
+    const NUM_TO_TEST: i32 = 10000;
+
+    let mut collection = database.get_collection(TEST_COLLECTION_NAME);
+    if database.has_collection(TEST_COLLECTION_NAME).unwrap() {
```

Comment:
In other tests I use `unwrap_or(())`. In that case you don't need to check, it just does nothing if it fails.

Reply:
I thought we wanted it to terminate on failure here?

Reply:
I think I placed this comment when there was still an if around this.